it/src/main/java/org/corfudb/universe/util/IpAddress.java

```diff
@@ -0,0 +1,17 @@
+package org.corfudb.universe.util;
+
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+
+@Builder
+@EqualsAndHashCode
+public class IpAddress {
```

Comment:
Why not just use InetSocketAddress?

Reply:
it just a typed wrapper for a string, we don't need InetSocketAddress here